lib/payload_handler/payload.rb

```diff
@@ -5,50 +5,77 @@ module PayloadHandler
   class Payload
     attr_reader :entities, :errors
 
+    def self.fetch(resource_types:, query_params: {})
```

Comment:
Does the usage of "resource_types:" represent a Map? Is this a sort of type casting in Ruby?